sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -267,6 +267,7 @@ object SQLConf {
       "Note: For structured streaming, this configuration cannot be changed between query " +
       "restarts from the same checkpoint location.")
     .intConf
+    .checkValue(_ > 0, "the value of sql.shuffle.partitions must be greater than 0")
```

Comment:
nit: spark.sql.shuffle.partitions?

Reply:
+1

Reply:
@WindCanDie, can you address this comment too?

Reply:
You can also write `s"... ${SHUFFLE_PARTITIONS.key} ..."`
Also capitalize "the"

Reply:
`"the value of sql.shuffle.partitions
${SHUFFLE_PARTITIONS.key} must be greater than 0"`